apps/src/code-studio/pd/professional_learning_landing/twoPartBanner.jsx

```diff
@@ -0,0 +1,60 @@
+/** Two section banner for the landing page */
+
+import React from 'react';
+
+const containerStyle = {
+  display: 'flex',
+  height: '240px',
+  width: '100%',
+};
+
+const imageStyle = {
+  backgroundSize: 'contain',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  backgroundColor: 'white',
+  height: '100%',
+  border: '1px solid black',
+  boxSizing: 'border-box',
+};
+
+const textStyle = {
+  fontSize: '24px',
+  backgroundColor: '#00adbc',
+  color: 'white',
+  padding: '20px'
+};
+
+const leftStyle = {
+  width: '50%',
+  maxWidth: '50%',
+  float: 'left',
+  borderTopLeftRadius: '10px',
+  borderBottomLeftRadius: '10px',
+};
+
+const rightStyle = {
+  width: '50%',
+  maxWidth: '50%',
+  float: 'left',
+  borderTopRightRadius: '10px',
+  borderBottomRightRadius: '10px',
+};
+
+const TwoPartBanner = React.createClass({
+  propTypes: {
+    leftHalf: React.PropTypes.element.isRequired,
+    rightHalf: React.PropTypes.element.isRequired
+  },
+
+  render() {
+    return (
+      <div style={containerStyle}>
+        {this.props.leftHalf}
+        {this.props.rightHalf}
+      </div>
+    );
+  }
+});
+
+export {TwoPartBanner, imageStyle, textStyle, leftStyle, rightStyle};
```

Comment:
See above. These styles shouldn't need to be exported. They can either be directly applied to the elements in props (leftHalf, rightHalf), or to container divs. In any case they should stay internal to this component.